src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,206 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class HtmlTitleTests extends NewTestTemplate {
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @DataProvider
+  private Object[][] dataHtmlTitleTest001() {
+    return new Object[][]{
+        // original title
+        {
+            "sktest123",
```

Comment:
You're right about the data separation. 
I'd like to say  that s/old_string/new_string/g feature considers all string occurences, no matter of their context. So it is possible to manipulate and rename the strings, but you may not be sure what is the context of these strings. If a string is assigned to variable, it's context, is defined and proper a name so that it is clear what that repeatable piece of data represents.